deploy-service/common/src/main/java/com/pinterest/deployservice/bean/HostBeanWithStatuses.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2016-2024 Pinterest, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.pinterest.deployservice.bean;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class HostBeanWithStatuses {
+    @JsonProperty("hostName")
+    private String host_name;
```

Comment:
since this is external facing, it can just be
```java
private String hostName;
```
The private field doesn't have to match the DB column name.